src/libraries/System.Private.CoreLib/src/System/Reflection/SignatureType.cs

```diff
@@ -99,7 +99,7 @@ public sealed override Type MakeArrayType(int rank)
         public sealed override string GetEnumName(object value) => throw new NotSupportedException(SR.NotSupported_SignatureType);
         public sealed override string[] GetEnumNames() => throw new NotSupportedException(SR.NotSupported_SignatureType);
         public sealed override Type GetEnumUnderlyingType() => throw new NotSupportedException(SR.NotSupported_SignatureType);
-        [RequiresDynamicCode("It might not be possible to create an array of the enum type at runtime. Use the GetEnumValues<TEnum> overload or the GetEnumValuesAsUnderlyingType method instead.")]
+        [RequiresDynamicCode("It might not be possible to create an array of the enum type at runtime. Use the Enum.GetValues<T> overload or the GetEnumValuesAsUnderlyingType method instead.")]
```

Comment:
```suggestion
        [RequiresDynamicCode("It might not be possible to create an array of the enum type at runtime. Use GetEnumValuesAsUnderlyingType or Enum.GetValues<T> instead.")]
```
Enum.GetValues is not overload of the annotated method.